### PR TITLE
[NayNay] Get all programs for an address

### PR DESCRIPTION
**Pull request by rh0delta**
## Description
- updated programs dev to add new method to fetch all programs for a given ss58 address
- renamed original get method to getProgramInfo
## GitHub Issue or Linear Task
Closes https://github.com/entropyxyz/cli/issues/66
## How Has This Been Tested?
used @JesseAbram address and returned programs attached
`Retrieved program pointers:
[
  '0x53b7dc00f5e4257be89bc2643eddc2c218623b047a2955e02181828efee3767f'
]`

### CODE REVIEW

File: src/programs/dev.ts [hunk 1]
```diff
@@ -48,14 +48,26 @@ export default class ProgramDev extends ExtrinsicBaseClass {
...
+  /**
+   * Retrieves all programs for a given address.
+   * 
+   * @param {string} address - SS58 Address
+   * @returns {Promise<string[]>} A promise that resolves to the list of program pointers
+   */
+
+  async get (address: string): Promise<any> {
+    const programs = await this.substrate.query.programs.ownedPrograms(address);
+    return programs
```
Comment (frankiebee) [inline src/programs/dev.ts:60]:
```suggestion
    return programs.toHuman()
```

**Review by frankiebee — CHANGES_REQUESTED** [review]
just a return type prefernce request otherwise looks good

**Comment by mixmix** [comment]
@rh0delta is this ready for re-review?

**Comment by rh0delta** [comment]
@mixmix yes it's ready for re-review, sorry thought Frankie was going to merge it in

**Review by frankiebee — APPROVED** [review]
LGTM

**Pull request merged**